lib/sync/index.js

```diff
@@ -89,38 +104,45 @@ function setLogLevel(dataset_id, params, cb) {
   }
 }
 
-function sync(dataset_id, params, callback) {
+function sync(dataset_id, params, cb) {
   syncUtil.doLog(dataset_id, 'info', 'sync');
 
   // TODO
 
   return cb(null, {});
 }
 
-/* Synchronise the individual records for a dataset */
-function syncRecords(dataset_id, params, callback) {
+// Synchronise the individual records for a dataset
+function syncRecords(dataset_id, params, cb) {
   syncUtil.doLog(dataset_id, 'info', 'syncRecords');
 
   // TODO
 
   return cb(null, {});
 }
 
+function connect(mongoDBConnectionUrl, redisUrl, cb) {
+  async.parallel([
+    function connectToMongoDB(callback) {
+      MongoClient.connect(mongoDBConnectionUrl, callback);
+    },
+    function connectToRedis(callback) {
+      var client = redis.createClient(redisUrl);
+      return callback(null, client);
+    }
+  ], function(err, results) {
+    if (err) return cb(err);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reformat the code to have only one statement per line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3AOneStatementPerLine)